src/main/webapp/app/core/interceptor/artemis-version.interceptor.ts

```diff
@@ -34,4 +59,49 @@ export class ArtemisVersionInterceptor implements HttpInterceptor {
             }),
         );
     }
+
+    /**
+     * Tells the service worker to check for updates and display an update alert if an update is available.
+     * This is either exactly if
+     * - the service worker detects an update right now, or
+     * - the first condition was ever true since the app loaded (aka last reload)
+     *
+     * We need to have this second option because the "checkForUpdate()" call sometimes starts to return false after a while, even though we didn't reload / update yet.
+     *
+     * @private
+     */
+    private checkForUpdates() {
+        // first update the service worker
+        this.updates
+            .checkForUpdate()
+            .then((updateAvailable: boolean) => {
+                if (this.hasSeenOutdatedInThisSession || updateAvailable) {
```

Comment:
I assume this is a typo
```suggestion
                if (this.hasBeenOutdatedInThisSession || updateAvailable) {
```